web/src/pages/index.tsx

```diff
@@ -1,14 +1,80 @@
-import { Heading } from '@chakra-ui/react'
+import { Image, Text, Heading, Box, SimpleGrid, Stack, Center } from "@chakra-ui/react"
 import { PageScaffold } from '../components/PageScaffold'
 import WithAuth, { WithAuthProps } from '../components/withAuth'
+import { LinkBoxHomePage } from "../components/LinkBoxHomePage";
 
-const Index: React.FC<WithAuthProps> = ({self}) => {
+const Index: React.FC<WithAuthProps> = ({ self }) => {
+
+  const getLinkBoxes = (userRole) => {
+    if (userRole === "COORDINATOR") {
+      return (
+        <SimpleGrid minChildWidth="325px" spacing="40px">
+          <LinkBoxHomePage bgc="#56CCF2" src="https://i.ibb.co/BjYJF0L/Tutor.png" height="44"
+            alt="Tutor-Bild" margTop="4" margBottom="unset" href="/tutorium"
+            text="Tutorium Management" /><LinkBoxHomePage bgc="#FC912A"
+              src="https://i.ibb.co/rtsrGTx/Sch-ler.png" height="44"
+              alt="Schüler-Bild" margTop="4" margBottom="unset" href="/"
+              text="Schüler Management" /><LinkBoxHomePage bgc="#94E43B"
+                src="https://i.ibb.co/GPvKhRF/Abwesenheit-buchen.png" height="44"
+                alt="Abwesenheit-Bild" margTop="4" margBottom="unset" href="/"
+                text="Abwesenheit buchen" /><LinkBoxHomePage bgc="#9B51E0"
+                  src="https://i.ibb.co/dDWNbHf/Support.png" height="48"
+                  alt="Support-Bild" margTop="unset" margBottom="-0.5" href="/"
+                  text="Support" /></SimpleGrid>
+      );
+    }
+
+    if (userRole === "TEACHER") {
+      return (
+        <Center><Stack direction={["column", "row"]} spacing="40px"><LinkBoxHomePage bgc="#94E43B"
+          src="https://i.ibb.co/GPvKhRF/Abwesenheit-buchen.png"
+          height="44" alt="Abwesenheit-Bild" margTop="4" margBottom="unset" href="/"
+          text="Abwesenheit buchen" /><LinkBoxHomePage bgc="#9B51E0"
+            src="https://i.ibb.co/dDWNbHf/Support.png" height="48"
+            alt="Support-Bild" margTop="unset" margBottom="-0.5" href="/"
+            text="Support" /></Stack></Center>
+      );
+    }
+  }
 
   return (
     <PageScaffold role={self.role}>
-
+      <Box w="full" right={0}>
+        <Box>
+          <SimpleGrid columns={[2, null, 3]} spacing="40px">
+            <Image
+              objectFit="cover"
+              src="https://image.jimcdn.com/app/cms/image/transf/none/path/se04d54dd5603e862/image/id9085589dffdbb35/version/1516615343/image.png"
```

Comment:
Pictures should be downloaded, so we don't rely on a third party to keep it available. On that note, could you try to replace pictures with React Icons/IconButtons, so we have a continuous art style throughout the website. 
You can get an overview of the available Icons [here](https://react-icons.github.io/react-icons/icons?name=fa)

Reply:
Done.

Reply:
could you try using only icons from [React icons FA (Font Awesome)](https://react-icons.github.io/react-icons/icons?name=fa) 
And I don't know about the others but for me all the different colors are a bit too much. What do you think, could one color for all be an option?

Reply:
Sry, but it is the logo of GeMont. So what should I do?

Reply:
That is different taste ^^.

Reply:
So all the icons from the Boxes are from react, but not the logo.

Reply:
I think one color for all LinkBoxes is not nice, in my opinion.